### ISSUE TITLE
Feature idea: Lambda Environment Variables set from deployment environment variables

### ISSUE THREAD

**Issue by madorb** [motdotla/node-lambda#321]
The `deploy.env` mechanism is fine, but doesn't work great for secrets, as the build/deploy process has to create/modify the file itself to add secret values. Would be great to have a mechanism to include values from the deploy-time environment to the runtime environment. Perhaps something like a convention such that all environment variables with a prefix are automatically deployed. for instance:

setting: 
`LAMBDA_NODE_DEPLOY_FOO=bar`

would be funcitonally equivalent of adding
`FOO=bar` 
to the `deploy.env` file. 

Don't want to go and try to build a PR for this if it's not something that would potentialy be accepted.

**Comment by DeviaVir** [comment]
I'm maybe misunderstanding, don't we already push all of them at the moment? https://github.com/motdotla/node-lambda/blob/master/lib/main.js#L185

**Comment by madorb** [comment]
@DeviaVir yes, you do... all of the ones specified in the deploy.env file... i'm suggesting a way to not have to create said file but instead being able to either a) provide the env variables directly on the command line, b) automagically add all env vars that start with a certain prefix

**Comment by DeviaVir** [comment]
@madorb Oh I see what you're saying. Yes, I could see that being useful. The command line utility could even go so far as to use an encryption helper, that would be awesome.

+1
